tests/unit_tests/test_wave_07.py

```diff
@@ -0,0 +1,185 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
```

Comment:
Fabulous work in this file! 💯 

Since we're not modifying any items here, we can also try to DRY this test file a bit and pull out the items to the top of the file and reuse them inside the tests! (However, we wouldn't want to do that if we ever had functionality that would mutate the items because it might have unintentional consequences for downstream tests!)